File: C++/1218.cpp
#include <iostream> 
#include <math.h>

using namespace std;

int main(){
	int a, b, c;
	cin >> a >> b >> c;
	
	if (a + b > c && a + c > b && b + c > a){
		if(a==b && b==c) cout << "정삼각형";
		else if(a==b || b==c || a==c) cout << "이등변삼각형";
		else if(pow(a, 2) + pow(b, 2) == pow(c, 2)) cout << "직각삼각형";
		else cout << "삼각형";
	} else{
		cout << "삼각형아님";
	}
}

File: Java/1218.java
import java.util.Scanner;
import java.io.*;

public class Main{
	public static void main(String[] args) throws Exception{
		Scanner sc = new Scanner(System.in);
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out, "utf-8")); // Exception처리를 안해주면 에러
		
		int a = sc.nextInt();
		int b = sc.nextInt();
		int c = sc.nextInt();
		
		if (a + b > c && a + c > b && b + c > a){
			if(a==b && b==c) bw.write("정삼각형");
			else if(a==b || b==c || a==c) bw.write("이등변삼각형");
			else if(Math.pow(a, 2) + Math.pow(b, 2) == Math.pow(c, 2)) bw.write("직각삼각형");
			else bw.write("삼각형");
		} else{
			bw.write("삼각형아님");
		}
		
		bw.close();
	}
}